DOCUMENTATION.md

```diff
@@ -104,751 +29,740 @@ import uk.gov.service.notify.SendSmsResponse;
 NotificationClient client = new NotificationClient(apiKey);
```

Comment:
The list of imports is not complete. Let's try only including the imports we need, which is `import uk.gov.service.notify.NotificationClient;`

Reply:
We may want to consider including the imports as needed in each snippet, but to be honest I don't think it's needed. The IDE java devs use do that for them.

Reply:
Deleted the unnecessary imports